docs/docs/faq.mdx

```diff
@@ -6,6 +6,6 @@ As Elf aims to be light and fast, we compile the code for modern browsers. Let u
 
 ## What About Akita?
 
-Akita is still actively maintained. Elf is merely another option.
+Akita is not actively maintained. Elf is merely another option.
```

Comment:
Maybe it should also say `Elf is the replacement`, if `Akita` is no longer maintained.

Reply:
thanks for the input. Actually I hope, that my PR is never merged :) it's more a discussion start on the future maintenance of Akita, since the Akita issues/PRs are being ignored

Reply:
and IMHO: for Elf to be a replacement of Akita, there should be a migration path